### ISSUE TITLE
ros config fails on fresh v0.4.0-rc11 install

### ISSUE THREAD

**Issue by TFenby** [rancher/os#614]
On a freshly-installed system, running the following, straight from the documentation, results in nothing being changed, and `ros config get` returns nothing:
```
$ sudo ros config set network.dns.domain myexampledomain.com
$ sudo ros config get network.dns.domain
```

**Comment by deniseschannon** [comment]
In v0.4.0, we moved to configuring a standard cloud config file so the command would need to have a prefix of rancher.

Sudo ros config set rancher.network.dns.domain myexample.com

**Comment by TFenby** [comment]
Still nothin'
```
[root@rancher cloud-config.d]# ros config set rancher.network.dns.domain myexample.com
[root@rancher cloud-config.d]# ros config get rancher.network.dns.domain
```

**Comment by deniseschannon** [comment]
Can you do a `ros config export` to see if the changes were saved?

**Comment by deniseschannon** [comment]
It looks like we removed `domain` from network as Docker doesn't support it. We'll need to update the docs for v0.4.0.

**Comment by deniseschannon** [comment]
Actually the docs for this change has already been submitted, but we haven't released the new docs for v0.4.0 as v0.4.0 isn't officially released yet.

**Comment by TFenby** [comment]
Ahh, that would make sense. Thanks for your help.
